bundles/org.openhab.binding.velbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -531,6 +537,25 @@
 		<config-description-ref uri="thing-type:velbus:33channelDeviceWithTemperatureSensor"/>
 	</thing-type>
 
+	<thing-type id="vmbelpir">
```

Comment:
README should be updated.
Default properties should be updated (velbus.properties) using the maven command.

Reply:
Can you give me the maven command to do the update of the default properties ?

Reply:
Changes done.